bug: cannot install this plugin

**Describe the bug**
Cannot install this plugin.
```
$ npm install @capacitor-community/privacy-screen
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: capacitor-plugin-demo@0.0.1
npm ERR! Found: @capacitor/core@2.4.7
npm ERR! node_modules/@capacitor/core
npm ERR!   @capacitor/core@"2.4.7" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"2.4.6" from @capacitor-community/privacy-screen@1.1.1
npm ERR! node_modules/@capacitor-community/privacy-screen
npm ERR!   @capacitor-community/privacy-screen@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install `@capacitor/core@2.4.7`
2. Run `npm install @capacitor-community/privacy-screen`
3. See error

**Expected behavior**
It should be possible to install this plugin with `@capacitor/core@2.4.7`.

--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
     "typescript": "3.8.3"
   },
   "peerDependencies": {
-    "@capacitor/core": "2.4.6"
+    "@capacitor/core": "^2.4.6"
   },
   "files": [
     "android/src/main/",

